reactDashboard/src/pages/Regional.jsx

```diff
@@ -47,6 +47,23 @@ const Regional = ({
 
   const currentRegionCode = useRef(regionCode);
   const currentLocation = useRef(match.url);
+  const [dateRange, setDateRange] = useState("dateRange");
+  const [maxDateRange, setMaxDateRange] = useState({
+    startDate: 0,
+    endDate: 0,
+  });
+
+  useEffect(() => {
+    // Only attempt to fetch data once
```

Comment:
The comment is no longer accurate - replace with: Get maximum date range to encompass all data (or similar) - or just bin the comment

Reply:
Binned